packages/build/src/log/description.ts

```diff
@@ -6,6 +6,7 @@ const BUILD_COMMAND_DESCRIPTIONS = {
   ui: 'Build command from Netlify app',
   config: 'build.command from netlify.toml',
   inline: 'build.command from a plugin',
+  heuristics: 'build.command detected through our heuristics',
```

Comment:
I wonder if this is a bit too technical? Could we say just "automatically detected"?

Reply:
sure!